docs/3/version-control_branching-merging.md

```diff
@@ -1,9 +1,10 @@
 # Branching & Merging
 
-Before we get into the specifics of how version control works lets examine two of the most import concepts, branching and merging.
+Before we get into the specifics of how version control works let's examine two of the most import concepts, branching and merging.
 
-![](img3/git-branches_light.svg ':size=600px :class=light-mode-img-center')
-![](img3/git-branches_dark.svg ':size=600px :class=dark-mode-img-center')
+<p style="text-align:center">
```

Comment:
It looks like your PR is rolling back some recent image changes. Please rebase or merge master into your branch and make sure it does not include unintended changes.